share/models/base.py

```diff
@@ -50,9 +51,10 @@ def __new__(cls, name, bases, attrs):
 
         version_attrs = {}
         for key, val in attrs.items():
-            if isinstance(val, models.Field) and val.unique:
+            if isinstance(val, models.Field) and (val.unique or val.db_index):
                 val = copy.deepcopy(val)
                 val._unique = False
+                val.db_index = False
```

Comment:
They're actually non existent. I add this in because one of them some how slipped through.